service/kvmconfig/v4/resource_set.go

```diff
@@ -16,6 +16,7 @@ import (
 
 	"github.com/giantswarm/kvm-operator/service/kvmconfig/v3/cloudconfig"
 	"github.com/giantswarm/kvm-operator/service/kvmconfig/v3/key"
+	"github.com/giantswarm/kvm-operator/service/kvmconfig/v3/resource/clusterrolebinding"
```

Comment:
This should be v4 I'm about to merge https://github.com/giantswarm/kvm-operator/pull/380

Reply:
It says now you have to resolve conflicts.